src/styles/theme.ts

```diff
@@ -162,6 +162,13 @@ const initTheme = (darkMode: boolean) => {
           sizeLarge: { fontSize: '16px' },
         },
       },
+      MuiLink: {
+        styleOverrides: {
+          button: ({ theme }) => ({
+            fontFamily: theme.typography.fontFamily,
```

Comment:
Can we not globally apply this to the entire theme?

Reply:
I think the component MuiLink overrides it

Reply:
In the [docs](https://mui.com/material-ui/react-link/#accessibility) they also apply `variant="body2"` to the Link and it overrides the native browser stylesheet thats causing the issue with font-family.

Reply:
Why do we have to override the font family for this component in particular while we have the global font family set?

https://github.com/safe-global/web-core/blob/main/src/styles/theme.ts#L68-L69

Wouldn't it be a bug in the material UI if it doesn't work just for this component?

Reply:
> overrides the native browser stylesheet thats causing the issue with font-family

The thing is, [we're MUI's `CSSBaseline`](https://github.com/safe-global/web-core/blob/main/src/pages/_app.tsx#L101).

Reply:
[We need to override the `CssBaseline` as well](https://mui.com/material-ui/customization/typography/#self-hosted-fonts).

Reply:
Even having the self-hosted fonts in the `MuiCssBaseline` override, the `button` font-family has to be overwritten it seems. Tried it in Chrome and Firefox.